evervault-inputs/src/main/java/com/evervault/sdk/input/ui/InlinePaymentCardInput.kt

```diff
@@ -12,10 +12,21 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.unit.dp
+import com.evervault.sdk.input.model.placeholder.PlaceholderDefaults
+import com.evervault.sdk.input.model.placeholder.PlaceholderTexts
+
+@Composable
+fun inlinePaymentCardInputLayout(): @Composable() (PaymentCardInputScope.(Modifier) -> Unit) =
+    { inline(modifier = it) }
+
+@Composable
+fun inlinePaymentCardInputLayout(placeholderTexts: PlaceholderTexts): @Composable() (PaymentCardInputScope.(Modifier) -> Unit) =
```

Comment:
a new method for providing `placeholderTexts` functionality to existing API users